migration/migrating-4-4/migrating-openshift-4_2-to-4.adoc

```diff
@@ -13,20 +13,19 @@ The CAM tool's web console and API, based on Kubernetes custom resources, enable
 .Prerequisites
 
 * You must have xref:../../cli_reference/openshift_cli/administrator-cli-commands.html#policy[`cluster-admin` privileges] on all clusters.
-* You must have a replication repository that supports the S3 API and is accessible to the source and target clusters.
+* You must configure a replication repository that supports the S3 API and is accessible to the source and target clusters.
```

Comment:
Change to:

* You must configure a replication repository that is accessible to the source and target clusters.